elrond_dex_pair/src/lib.rs

```diff
@@ -1022,15 +976,19 @@ pub trait Pair {
         self.state().get() == State::Active
     }
 
+    #[view]
+    fn getLpTokenIdentifier(&self) -> TokenIdentifier {
```

Comment:
You can put this view function directly on the storage mapper

Reply:
Unfortunately, you cannot